articles/search/index-add-suggesters.md

```diff
@@ -103,6 +103,15 @@ Properties that define a suggester include the following:
 |`searchMode`  |The strategy used to search for candidate phrases. The only mode currently supported is `analyzingInfixMatching`, which performs flexible matching of phrases at the beginning or in the middle of sentences.|
 |`sourceFields`|A list of one or more fields that are the source of the content for suggestions. Only fields of type `Edm.String` and `Collection(Edm.String)` may be sources for suggestions. Only fields that don't have a custom language analyzer set can be used.<p/>Specify only those fields that lend themselves to an expected and appropriate response, whether it's a completed string in a search bar or a dropdown list.<p/>A hotel name is a good candidate because it has precision. Verbose fields like descriptions and comments are too dense. Similarly, repetitive fields, such as categories and tags, are less effective. In the examples, we include "category" anyway to demonstrate that you can include multiple fields. |
 
+#### Analysis of SourceFields in a suggester
+
+Azure Search analyzes the field content to enable querying on individual terms. There is support for various pre-built and custom configurations to customize the analysis process. Suggesters require prefixes (edge-ngrams) to be stored in addition to the terms for incomplete matches. This is achieved by doing additional analysis over the source fields.
```

Comment:
I believe this is too nuanced and unnecessarily exposes some implementation details that might change and users shouldn't need to understand. I think I'd be enough to say that currently custom analyzers are not supported on fields with suggesters enabled because suggesters assume complete terms are indexed and custom analyzers allow to break that assumption. We'll do work in the future to allow terms to be normalized with a limited set of analysis components now available though the Custom Analyers API butsome of them will always remain incompatible with the internal document and query processing pipeline that supports Suggestions and Autocomplete.